Makefile

```diff
@@ -483,7 +483,7 @@ endif
 run-db-container:
 	docker ps -q --filter "name=postgres" | xargs -r docker kill && sleep 3
 	docker run -d  --rm --tmpfs /var/lib/pgsql/data --name postgres -e POSTGRESQL_ADMIN_PASSWORD=admin -e POSTGRESQL_MAX_CONNECTIONS=10000 -p 127.0.0.1:5432:5432 \
-	quay.io/edge-infrastructure/postgresql-12-centos7:latest
+		$(PSQL_IMAGE)
```

Comment:
nit: it looks like there is extra indentation here

Reply:
it's on purpose, so that people can tell it's a continuation of the previous line